spec/models/thermometer_widget_spec.rb

```diff
@@ -1,41 +1,25 @@
 describe ThermometerWidget do
 
-  let(:content) { {
+  let(:params) { {
     goal: 10000,
     count: 500,
-    autoincrement: true
+    autoincrement: true,
+    page_display_order: 1
   } }
-  let(:string_content) { {
-    goal: "12345",
-    count: "45",
-    autoincrement: "0"
-  } }
```

Comment:
PG hstore only stores key / values as strings. But, it's smart enough to know when a "4000" should be a 4000 - I think. I'll check the API, again.